iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -1162,6 +1171,32 @@ IOTHUB_CLIENT_RESULT IoTHubClient_LL_UploadToBlob_SetOption(IOTHUB_CLIENT_LL_UPL
             upload_data->blob_upload_timeout_secs = *(size_t*)value;
             result = IOTHUB_CLIENT_OK;
         }
+        else if (strcmp(optionName, OPTION_CURL_INTERFACE) == 0)
```

Comment:
Please submit a PR for the OPTION_CURL_INTERFACE in the Azure/azure-c-shared-utility. Then you need to update the submodule in this repo.

Reply:
done.